app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -0,0 +1,68 @@
+package com.example.ithaca_transit_android_v2.states
+
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+
+// Parent Class
+sealed class SearchState(
+    val startLocation: Location = Location("", Coordinate(0.0,0.0),""),
+    val endLocation: Location = Location("", Coordinate(0.0,0.0),"")
```

Comment:
You're missing a space after the last comma in this line and the above, should be `, "")` 

Do you have a linter set up, if not ask @ningning621!

Reply:
`CTRL + Shift + Alt + L`!